### PR TITLE
feature/fgprof

### PR DESCRIPTION
**Pull request by zstone12**
#### What this PR does / why we need it (English/Chinese):
add fgprof for hertz.

#### Which issue(s) this PR fixes:

TODO:
- [x]  Add unit tests for fgprof
- [ ] Add information about fgprof to the README document

### CODE REVIEW

**Comment by li-jin-gou** [comment]
❤️

File: pprof_test.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 CloudWeGo Authors
```
Comment (welkeyever) [inline pprof_test.go]: No need update all - only modified file

File: adaptor/handler.go [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 CloudWeGo Authors
+ * Copyright 2023 CloudWeGo Authors
```
Comment (welkeyever) [inline adaptor/handler.go]: AA

File: adaptor/handler_test.go [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2022 CloudWeGo Authors
+ * Copyright 2023 CloudWeGo Authors
```
Comment (welkeyever) [inline adaptor/handler_test.go]: AA

File: pprof_test.go (hunk 1 above)
Comment (zstone12) [reply pprof_test.go]: get

Comment (zstone12) [reply pprof_test.go]: When I changed it back to 2022, the corresponding file failed the CI test (which is why I changed the existing file to 2023 in the first place, I checked `.licenserc.yaml` and it seemed fine).This makes me feel a little confused.
img_v2_00a6bdb3-99e5-4d2d-8ddf-e367375af6eg

Comment (welkeyever) [reply pprof_test.go]: https://github.com/cloudwego/hertz/blob/develop/.github/workflows/pr-check.yml#L12 
Modify this to align the hertz main library and it should be fine.

Comment (zstone12) [reply pprof_test.go]: Got it.thanks

**Review by welkeyever — APPROVED** [review]
LGTM, thx~

**Review by li-jin-gou — APPROVED** [review]
LGTM

**Pull request merged**